src/webviews/webview-side/interactive-common/common.css

```diff
@@ -10,8 +10,8 @@ html {
 }
 
 #main-panel {
-    background: var(--override-background, var(--vscode-editor-background));
-    color: var(--override-foreground, var(--vscode-editor-foreground));
+    background: var(--vscode-editor-background);
```

Comment:
Do we need this? I would have thought this is not necessary.

Reply:
Doesn't matter

Reply:
Just trying to make the least changes possible, `--vscode-editor-foreground` is in many places 😅